lastuser_oauth/forms/profile.py

```diff
@@ -51,7 +51,7 @@ def validate_old_password(self, field):
 
 class ProfileForm(forms.Form):
     fullname = forms.StringField(__("Full name"),
-        validators=[forms.validators.DataRequired(), forms.validators.Length(max=80)])
+        validators=[forms.validators.DataRequired(), forms.validators.Length(max=63)])
```

Comment:
Why the length restriction on full name? Username is created from it?

Reply:
DNS labels are restricted to 63 characters. Any longer and Talkfunnel's subdomains will break.

Reply:
No I know that and get the limit for username. full name is never used on url, right?

Reply:
Whoops, validator needed to be on the other field.